dev-client/src/components/buttons/DismissButton.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2023 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {StyleSheet, Text} from 'react-native';
+
+import {Button} from 'native-base';
+
+import {TranslatedContent} from 'terraso-mobile-client/components/content/typography/TranslatedContent';
+import {convertColorProp} from 'terraso-mobile-client/components/util/nativeBaseAdapters';
+
+export default function DismissButton() {
+  return (
+    <Button
+      borderColor="alert.errorContent"
+      _pressed={{backgroundColor: 'transparent'}} // TODO: get actual color
```

Comment:
Do we have the actual color now? Is that what `action.selected` in theme.ts is?

Reply:
I confirmed with Courtney that the button should remain transparent when selected. I'll remove the TODO.